datafusion/physical-expr/src/datetime_expressions.rs

```diff
@@ -1308,6 +1349,40 @@ fn validate_to_timestamp_data_types(
     None
 }
 
+// TODO: 实现这个函数
+/// to_date SQL function implementation
+pub fn to_date_invoke(args: &[ColumnarValue]) -> Result<ColumnarValue> {
+    if args.is_empty() {
+        return internal_err!(
```

Comment:
```suggestion
        return exec_err!(
```

Reply:
it seems that other functions are all using internal_err. If i need to change all of them？

Reply:
Based on https://github.com/apache/arrow-datafusion/pull/8781#discussion_r1444685792, I vote for `exec_err`

Reply:
So need I change   `internal_err` in  `to_timestamp_invoke`  to `exec_err`?
Or it should be done in another PR?

Reply:
A follow on pr is better